textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinTest.java

```diff
@@ -1,2015 +0,0 @@
-                } else if (index == 3) {
-                    IField[] book2_3 = { new IntegerField(63), new StringField("Paul Kalanithi"),
-                            new StringField("When Breath Becomes Air"), new IntegerField(256),
-                            new TextField("Review of a Book. This is a typical "
-                                    + "review. This is a test. A book review " + "test. A test to test queries without "
-                                    + "actually using actual review. From " + "here onwards, we can pretend this to "
-                                    + "be actually a review even if it is not " + "your typical book review.") };
-                    index--;
-                    tupleArray[index] = new DataTuple(schema, book2_3);
-                } else if (index == 2) {
-                    IField[] book2_2 = { new IntegerField(64), new StringField("Matthew Desmond"),
-                            new StringField("Evicted: Poverty and Profit in the " + "American City"),
-                            new IntegerField(432),
-                            new TextField("Review of a Book. This is a typical "
-                                    + "review. This is a test. A book review " + "test. A test to test queries without "
-                                    + "actually using actual review. From " + "here onwards, we can pretend this to "
-                                    + "be actually a review even if it is not " + "your typical book review.") };
-                    index--;
-                    tupleArray[index] = new DataTuple(schema, book2_2);
-                } else if (index == 1) {
-                    IField[] book2_1 = { new IntegerField(65), new StringField("Sharon Guskin"),
-                            new StringField("The Forgetting Time: A Novel"), new IntegerField(368),
-                            new TextField("Review of a Book. This is a typical "
-                                    + "review. This is a test. A book review " + "test. A test to test queries without "
-                                    + "actually using actual review. From " + "here onwards, we can pretend this to "
-                                    + "be actually a review even if it is not " + "your typical book review.") };
-                    index--;
-                    tupleArray[index] = new DataTuple(schema, book2_1);
-                }
-            }
-            break;
-
-        default:
-            break;
-        }
-
-        return Arrays.asList(tupleArray);
-    }
-
-    // This method cleans up after each test.
-    @After
-    public void cleanUp() throws Exception {
-        dataWriterForOuter.clearData();
-        dataWriterForInner.clearData();
-    }
-
-    // This case tests for scenario when the IDs of the documents don't match.
-    // Test result: The list of result returned is empty.
-    @Test
-    public void testIdsDontMatch() throws Exception {
```

Comment:
ALL the test cases are migrated using the new helper functions. The test logic remains the same.